### PR TITLE
Bug fix for the Amazon Pinpoint - Abandoned Cart Campaign section.

### PR DESCRIPTION
**Pull request by aonz**
*Issue #, if available:*

Amazon Pinpoint returns the error 400 response when clicking on the "Trigger Abandoned Cart email" button.

*Description of changes:*

Remove the unnecessary attribute causing Amazon Pinpoint to throw the error and update the Jupyter notebook with the correct event name.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by alexchirayath — APPROVED** [review]
Looks good to me. Please wait until Bastien/James can merge the commit it and update the staging for the repo

**Comment by james-jory** [comment]
Looks good to me too.

**Pull request merged**
